hudi-io/src/main/java/org/apache/hudi/storage/inline/InLineFSUtils.java

```diff
@@ -59,42 +53,13 @@ public static StoragePath getInlineFilePath(StoragePath outerPath,
                                               long inLineLength) {
     final String subPath = new File(outerPath.toString().substring(outerPath.toString().indexOf(":") + 1)).getPath();
     return new StoragePath(
-        InLineFileSystem.SCHEME + SCHEME_SEPARATOR
+        SCHEME + SCHEME_SEPARATOR
             + StoragePath.SEPARATOR + subPath + StoragePath.SEPARATOR + origScheme
             + StoragePath.SEPARATOR + "?" + START_OFFSET_STR + EQUALS_STR + inLineStartOffset
             + "&" + LENGTH_STR + EQUALS_STR + inLineLength
     );
   }
 
-  /**
-   * InlineFS Path format:
-   * "inlinefs://path/to/outer/file/outer_file_scheme/?start_offset=start_offset>&length=<length>"
-   * <p>
-   * Outer File Path format:
-   * "outer_file_scheme://path/to/outer/file"
-   * <p>
-   * Example
-   * Input: "inlinefs://file1/s3a/?start_offset=20&length=40".
-   * Output: "s3a://file1"
-   *
-   * @param inlineFSPath InLineFS Path to get the outer file Path
-   * @return Outer file Path from the InLineFS Path
-   */
-  public static Path getOuterFilePathFromInlinePath(Path inlineFSPath) {
```

Comment:
Maybe we can get rid of these utils taking Hadoop `Path` for InlineFS and reuse the utils with `StoragePath`?